doc/release-notes/release-notes-bucash1.5.1.0-rc1.md

```diff
@@ -0,0 +1,99 @@
+Release Notes for Bitcoin Unlimited Cash Edition 1.5.1.0-rc1
+=========================================================
+
+Bitcoin Unlimited Cash Edition release candidate 1 for version 1.5.1.0 is now available from:
+
+  <https://bitcoinunlimited.info/download>
+
+Please report bugs using the issue tracker at github:
+
+  <https://github.com/BitcoinUnlimited/BitcoinUnlimited/issues>
+
+This is a main release candidate based of Bitcoin Unlimited compatible
+with the Bitcoin Cash specifications you could find here:
+
+- https://github.com/bitcoincashorg/bitcoincash.org/blob/master/spec/uahf-technical-spec.md (Aug 1st '17 Protocol Upgrade, bucash 1.1.0.0)
+- https://github.com/bitcoincashorg/bitcoincash.org/blob/master/spec/nov-13-hardfork-spec.md (Nov 13th '17 Protocol Upgrade, bucash 1.1.2.0)
+- https://github.com/bitcoincashorg/bitcoincash.org/blob/master/spec/may-2018-hardfork.md (May 15th '18 Protocol Upgrade, bucash 1.3.0.0, 1.3.0.1, 1.4.0.0)
+- https://github.com/bitcoincashorg/bitcoincash.org/blob/master/spec/2018-nov-upgrade.md (Nov 15th '18 Protocol Upgrade, bucash 1.5.0.0, 1.5.0.1, 1.5.0.2)
+
+This release candidate is compatible with both [Bitcoin Cash](https://github.com/bitcoincashorg/bitcoincash.org/blob/master/spec/2018-nov-upgrade.md) and [SV](https://github.com/bitcoin-sv/bitcoin-sv/blob/master/doc/release-notes.md) changes to the consensus rules.
+SV features set is **disabled by default**, the default policy is to activate the set of changes as defined by the bitcoincash.org.
+
+Although this software has passed normal QA tests, it is currently undergoing longevity testing to validate stability.  If you are looking for the most stable software, use the last Official Release (i.e. 1.5.0.2).
+
+Upgrading
+---------
+
+If you are running an older version, shut it down. Wait until it has completely
+shut down (which might take a few minutes for older versions), then run the
+installer (on Windows) or just copy over /Applications/Bitcoin-Qt (on Mac) or
+bitcoind/bitcoin-qt (on Linux).
+
+If you are upgrading from a release older than 1.1.2.0, your UTXO database will be converted
+to a new format. This step could take a variable amount of time that will depend
+on the performance of the hardware you are using.
+
+Other than that upgrading from a version lower than 1.3.0.0 your client is probably stuck
```

Comment:
update 1.3.0.0 to 1.5.0.1 since that was the first client supporting this round of HFs

Reply:
done